samples/glean/src/main/java/org/mozilla/samples/glean/GleanApplication.kt

```diff
@@ -20,6 +20,7 @@ class GleanApplication : Application() {
         // Initialize the Glean library. Ideally, this is the first thing that
         // must be done right after enabling logging.
         Glean.initialize(applicationContext)
+        Glean.setMetricsEnabled(false)
```

Comment:
Why do we want/need this?

Reply:
We don't it shouldn't be there, it was for testing something and I neglected to filter it out when I committed.